modules/channel-web/src/backend/db.ts

```diff
@@ -367,6 +366,26 @@ export default class WebchatDb {
     })
   }
 
+  async deleteConversation(userId: string, conversationId: string, botId: string) {
+    return this.knex.transaction(async trx => {
+      await trx('events')
+        .del()
+        .where({ threadId: conversationId })
+
+      await trx('web_messages')
+        .del()
+        .where({ conversationId })
+
+      await trx('web_conversations')
+        .del()
+        .where({ id: conversationId, userId, botId })
+
+      await trx('srv_channel_users')
+        .del()
+        .where({ user_id: userId, channel: 'web' })
+    })
```

Comment:
Please always include the userId, threadId AND botId when deleting from any table in the database. Those are too broad, technically you could pick any conversation ID and is not a totally reliable ID.